templates/case-studies/templates/_macros-image.jinja

```diff
@@ -0,0 +1,18 @@
+# Params
+# image_url: URL of image (required)
+# caption: Caption for image
+
+{%- macro image(image_url, caption) -%}
+  <section class="p-strip">
+    <div class="u-fixed-width">
+      <div class="row">
+        <div class="u-align--center u-vertically-center u-no-padding--bottom p-image-container--cinematic">
```

Comment:
```suggestion
      <div class="row">
        <div class="col-12 u-align--center u-vertically-center u-no-padding--bottom p-image-container--cinematic">
```

Reply:
`u-fixed-width` accomplishes the same as `row`, so can be omitted here. But make sure you wrap children of a `row` in a `col`.

Another option would be to wrap the image in its own `u-fixed-width` and the caption in a `row > col`

Reply:
Thanks for the catch, I'll remove `u-fixed-width` since it is redundant in this case